src/edu/stanford/nlp/ie/NERFeatureFactory.java

```diff
@@ -1747,78 +1833,74 @@ else if (flags.useTitle2) {
-          featuresCpC.add("P_ORDINAL");
+          out.add("P_ORDINAL");
         }
       }
 
       if (flags.useAbbr || flags.useMinimalAbbr) {
-        featuresCpC.add(p.get(CoreAnnotations.AbbrAnnotation.class) + '-' + c.get(CoreAnnotations.AbbrAnnotation.class) + "-PABBRANS");
+        out.addConcat(p.get(CoreAnnotations.AbbrAnnotation.class), "-", c.get(CoreAnnotations.AbbrAnnotation.class), "-PABBRANS");
       }
 
       if (flags.useAbbr1 || flags.useMinimalAbbr1) {
         if (!c.get(CoreAnnotations.AbbrAnnotation.class).equals("XX")) {
-          featuresCpC.add(p.get(CoreAnnotations.AbbrAnnotation.class) + '-' + c.get(CoreAnnotations.AbbrAnnotation.class) + "-PABBRANS");
+          out.addConcat(p.get(CoreAnnotations.AbbrAnnotation.class), "-", c.get(CoreAnnotations.AbbrAnnotation.class), "-PABBRANS");
         }
       }
 
       if (flags.useChunkySequences) {
-        featuresCpC.add(p.get(CoreAnnotations.ChunkAnnotation.class) + '-' + c.get(CoreAnnotations.ChunkAnnotation.class) + '-' + n.get(CoreAnnotations.ChunkAnnotation.class) + "-PCNCHUNK");
+        out.add(p.get(CoreAnnotations.ChunkAnnotation.class) + '-' + c.get(CoreAnnotations.ChunkAnnotation.class) + '-' + n.get(CoreAnnotations.ChunkAnnotation.class) + "-PCNCHUNK");
       }
 
       if (flags.usePrev) {
         if (flags.useSequences && flags.usePrevSequences) {
-          featuresCpC.add("PSEQ");
-          featuresCpC.add(cWord + "-PSEQW");
+          out.add("PSEQ");
+          out.addConcat(cWord, "-PSEQW");
 
           if ( ! flags.strictGoodCoNLL) {
-            featuresCpC.add(pWord+ '-' +cWord + "-PSEQW2");  // added later after goodCoNLL
-            featuresCpC.add(pWord + "-PSEQpW"); // added later after goodCoNLL
+            out.addConcat(pWord, "-", cWord, "-PSEQW2");  // added later after goodCoNLL
+            out.addConcat(pWord, "-PSEQpW"); // added later after goodCoNLL
           }
 
           if (flags.useDistSim) {
-            featuresCpC.add(pDS + "-PSEQpDS");
-            featuresCpC.add(cDS + "-PSEQcDS");
-            featuresCpC.add(pDS+ '-' +cDS + "-PSEQpcDS");
+            out.addConcat(pDS, "-PSEQpDS");
+            out.addConcat(cDS, "-PSEQcDS");
+            out.addConcat(pDS, "-", cDS, "-PSEQpcDS");
           }
 
           if (((flags.wordShape > WordShapeClassifier.NOWORDSHAPE) || flags.useShapeStrings)) {
             if ( ! flags.strictGoodCoNLL) {     // These ones were added later after goodCoNLL
-              featuresCpC.add(pShape + "-PSEQpS");
-              featuresCpC.add(cShape + "-PSEQcS");
-            }
-            if (flags.strictGoodCoNLL && ! flags.removeStrictGoodCoNLLDuplicates) {
-              featuresCpC.add(pShape + '-' + cShape + "-PSEQpcS"); // Duplicate (in goodCoNLL orig, see -TYPES below)
+              out.addConcat(pShape, "-PSEQpS");
+              out.addConcat(cShape, "-PSEQcS");
             }
+            if (flags.strictGoodCoNLL && ! flags.removeStrictGoodCoNLLDuplicates)
```

Comment:
The diff is a lot easier to read if there are fewer {} changes mixed in with the actual code changes :)

Reply:
Yes, these got worse over time as the branches diverged I think. It would be easier if there were a consistent code style everywhere; such as always using {}. In my other pull request, I spend some hours yesterday trying to remove as many of such changes as I could find.